### ISSUE TITLE
Move from toot to post

### ISSUE THREAD

**Issue by koyuawsmbrtn** [nolanlawson/pinafore#2023]
Pinafore can also be used with other fediverse server implementations such as Pleroma and therefore it'd be nice if the toots would be called posts and the "TOOT" button should be chaned to "POST".

**Comment by nolanlawson** [comment]
Thanks for opening the issue! Pinafore indeed works with non-Mastodon fediverse services. However, it's primarily designed as a Mastodon client. Mastodon calls them "toots," so Pinafore also calls them "toots."

If you'd like to change this, you can modify [the English language file](https://github.com/nolanlawson/pinafore/blob/081df2bf826fb419d10f9d0d62d30c2d3469b3e9/src/intl/en-US.js) and replace all instances of "toot" with "post," and host Pinafore on your own server. Thanks for understanding!

**Comment by davidbtc2009** [comment]
I believe mastodon has now changed this officially.

**Comment by nolanlawson** [comment]
Yeah I know, I still like "toot" personally, but I guess the times they are a-changin'.
